test/vstest.console.UnitTests/vstest.console.UnitTests.csproj

```diff
@@ -19,7 +19,7 @@
       <FromP2P>true</FromP2P>
     </ProjectReference>
   </ItemGroup>
-  <ItemGroup>
+  <ItemGroup Condition=" '$(OS)' == 'Windows_NT' OR '$(TargetFramework)' != 'net451' ">
```

Comment:
This is bleh, but when I just enable it it succeeds on windows but not on linux, when I make it conditional to just NOT net451 it fails on windows. And without the ref it fails on Linux...